local/compose/build.go

```diff
@@ -187,6 +187,18 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opts
 	if err == nil {
 		err = errW
 	}
+
+	for imageName := range opts {
+		for _, c := range observedState {
+			if c.Image == imageName {
+				err := s.restartContainer(ctx, c)
```

Comment:
Are we sure we don't need to recreate the container, to use the updated image ? (this is invoking `s.apiClient.ContainerRestart()` here)